src/vdxf/index.ts

```diff
@@ -65,9 +67,9 @@ export class VDXFObject implements VDXFObjectInterface {
   toDataBuffer() {
     return Buffer.alloc(0);
   }
-  
+
   fromDataBuffer(buffer: Buffer, offset: number = 0) {
-    return offset;
+    return offset + 1;
```

Comment:
This is a placeholder method, meant to always be overwritten with a custom fromDataBuffer function, unsure why the offset is changed.